mod/reportedcontent/classes/Elgg/ReportedContent/Notifications/CreateReportedContentNotificationHandler.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Elgg\ReportedContent\Notifications;
+
+use Elgg\Notifications\NotificationEventHandler;
+use Elgg\Database\QueryBuilder;
+
+class CreateReportedContentNotificationHandler extends NotificationEventHandler {
+	
+	/**
+	 * {@inheritDoc}
+	 */
+	public function getSubscriptions(): array {
+		$admins = elgg_get_entities([
+			'type' => 'user',
+			'limit' => false,
+			'batch' => true,
+			'metadata_name_value_pairs' => [
+				[
+					'name' => 'admin',
+					'value' => 'yes',
+				],
+			],
+			'private_setting_name_value_pairs' => [
+				[
+					'name' => 'admin_notification',
+					'value' => 1,
+				],
+			],
+			'private_setting_name_prefix' => 'plugin:user_setting:reportedcontent:',
+			'wheres' => [
+				function (QueryBuilder $qb, $main_alias) {
+					// no idea why an admin would report content and not fix it themselfs, but just in case
+					return $qb->compare("{$main_alias}.guid", '!=', $this->event->getActorGUID(), ELGG_VALUE_GUID);
```

Comment:
maybe this check is 'cheaper' in the next foreach loop??

Reply:
I think not